src/components/App/index.spec.tsx

```diff
@@ -30,15 +32,11 @@ describe(__filename, () => {
       },
     };
 
-    // TODO: Use shallowUntilTarget()
-    // https://github.com/mozilla/addons-code-manager/issues/15
-    const root = shallow(<App authToken={authToken} />, context)
-      // withRouter HOC
-      .shallow(context)
-      // connect HOC
-      .shallow(context)
-      // base component
-      .shallow();
+    const root = shallowUntilTarget({
+      componentInstance: <App authToken={authToken} />,
+      targetComponent: AppBase,
```

Comment:
I would prefer to keep the old signature, we don't need object properties for the instance and the target component.